ipaserver/install/server/__init__.py

```diff
@@ -177,6 +176,9 @@ class ServerInstallInterface(ServerCertificateInstallInterface,
     preserve_sssd = False
     no_sssd = False
 
+    force_join = client.ClientInstallInterface.force_join
+    force_join = replica_install_only(force_join)
```

Comment:
Right. I meant to provide a more general approach but it probably does not make much sense here.